src/components/tl/ucp/tl_ucp_ep.c

```diff
@@ -51,47 +51,52 @@ ucc_status_t ucc_tl_ucp_connect_team_ep(ucc_tl_ucp_team_t         *team,
     return ucc_tl_ucp_connect_ep(ctx, ep, addr);
 }
 
+static inline ucp_ep_h get_next_ep(ucc_tl_ucp_context_t *ctx, int *i)
```

Comment:
I'm not sure get_next_ep should be a function, I think better move code inside close_eps because 
1. Where else we can use that function?
2. It does different things for different types of storage, for array it just returns next ep but for hash it additionally removes ep from storage

Reply:
it is used twice in the ucc_tl_ucp_close_eps fn: i didn't want to duplicate it. It does different things indeed and only used for eps close flow. Will it be better if i change "static inline" to helper macro?